harmonica/synthetic/surveys.py

```diff
@@ -0,0 +1,159 @@
+"""
+Create synthetic surveys for gravity and magnetic observations
+"""
+from verde import get_region, inside
+from verde.coordinates import check_region
+
+from ..datasets import fetch_britain_magnetic, fetch_south_africa_gravity
+
+
+def airborne_survey(region=None, data_region=(-5.0, -4.0, 56.0, 56.5)):
+    """
+    Create measurement locations for a synthetic airborne survey.
+
+    The observation points are sampled from the Great Britain total-field magnetic
+    anomaly dataset (see :func:`harmonica.datasets.fetch_britain_magnetic`).
+    A portion of the original survey is cut (*data_region*) and the coordinates may be
+    scaled to the given *region*.
```

Comment:
One thing to remember now is to crop these lines at 79 characters.

Reply:
You're right. I added it to my TODO list, but forgot to do so.

Reply:
By the way, is there a quick way to do this in vim? If it is, we should include it on the Contributing Guidelines. It's not an automatic way to do so, but at least it's better than nothing.
For example, I managed to do it by entering into VISUAL mode, select the lines I wanted to shrink and used `gq`. Nevertheless, I needed to select each block of the docstring separately instead of selecting the entire docstring and doing it only once.

Reply:
Good question. I did it the same way you did. You could write a macro but it might be more work than doing it manually.

Reply:
I think we will have to do it manually then. Another option is to propose this kind of automation on Black.

Reply:
The problem then would be to have black use different widths for docstrings and code. Which I doubt they would allow :(